.github/workflows/autoformat.yml

```diff
@@ -93,6 +93,7 @@ jobs:
       - uses: ./.github/actions/setup-python
       - run: |
           pip install -r requirements/lint-requirements.txt
+          cargo install taplo-cli@0.9.0 --locked
```

Comment:
https://github.com/toml-lang/toml/issues/532

Reply:
Chose https://github.com/tamasfe/taplo because it seems maintained has > 1K stars.

Reply:
Is there a way to install this or any other formatter without cargo? It might be a bit weird that MLflow development needs cargo while it doesn't depend on Rust (ruff use it but it can be installed via pip).

Reply:
Yes, you can use NPM or homebrew. Chose cargo here for performance.

Reply:
I think pre-built binaries will be available in the next release:

https://github.com/tamasfe/taplo/issues/542